forum/projects/utils.py

```diff
@@ -0,0 +1,15 @@
+def get_changed_fields(old_instance, new_instance):
+    """
+    Function compare project data and return changed
+    """
+    changes = {}
+    for field in old_instance._meta.fields:

```

Comment:
Ensure both instances (`old_instance` and `new_instance`) are of the same model class

Reply:
done!